web/src/pm_conversations.js

```diff
@@ -31,6 +31,28 @@ class RecentDirectMessages {
     recent_message_ids = new FoldDict(); // key is user_ids_string
     recent_private_messages = [];
 
+    removeEmptyConvo() {
+        const convoList = this.recent_private_messages;
+        for (let i = 0; i < convoList.length; i += 1) {
+            if (!convoList[i].max_message_id) {
+                convoList.splice(i, i + 1);
+                break;
+            }
+        }
+    }
+
+    initInsert(user_ids) {
+        const user_ids_string = user_ids.join(",");
+        const conversation = {
+            user_ids_string,
+            max_message_id: null,
+        };
+        // adds to top of recent private messages list, if not already present.
+        if (!this.recent_private_messages.some((obj) => obj.user_ids_string === user_ids_string)) {
```

Comment:
Let's use `includes` instead of `some` since it is faster.

Reply:
done